docs/reference/commandline/dockerd.md

```diff
@@ -130,8 +130,11 @@ to [the `daemon.json` file](#daemon-configuration-file).
 
 ### Environment variables
 
-For easy reference, the following list of environment variables are supported
-by the `dockerd` command line:
+The following list of environment variables are supported by the `dockerd` daemon.
+Some of these environment variables are supported both by the Docker Daemon and
+the `docker` CLI. Refer to the [Environment variables section](cli.md#environment-variables)
+in the CLI reference to learn about environment variables supported by the
```

Comment:
This page is no longer under the CLI reference in the TOC, moved it to engine > manuals. https://docs.docker.com/engine/reference/commandline/cli/#environment-variables

```suggestion
in the CLI section to learn about environment variables supported by the
```

Reply:
I changed it to this (tried to avoid `section` twice in the wording); https://github.com/docker/cli/compare/b7c481624460d19f0970126b1f818806e4141000..79992184e0d4cdaa08f6920b649769c77cdd0558

<img width="734" alt="Screenshot 2024-01-11 at 14 56 29" src="https://github.com/docker/cli/assets/1804568/a599f783-9d8f-4151-a36a-9a672cb5e139">